data/json/monsters/zed_burned.json

```diff
@@ -65,9 +65,9 @@
     "aggression": 100,
     "morale": 100,
     "melee_skill": 4,
-    "melee_dice": 1,
-    "melee_dice_sides": 10,
-    "melee_cut": 14,
+    "melee_dice": 3,
+    "melee_dice_sides": 5,
+    "melee_cut": 10,
```

Comment:
What are you comparing it to here? One of the skeletons?

Reply:
i was comparing it to the baseline zombie brute, and its cut seemed a bit too severe so i switched a bit over to bash

Reply:
might be better comparably to make it two sides and 8 damage, to better line up with brutes stronger hits